frontend/public/components/utils/kebab.tsx

```diff
@@ -207,7 +207,9 @@ export type KebabOption = {
   href?: string, callback?: () => any;
   accessReview?: AccessReviewResourceAttributes;
 };
-export type KebabAction = (kind: K8sKind, obj: K8sResourceKind) => KebabOption;
+export type KebabAction = (kind: K8sKind, obj: K8sResourceKind, resources?: any) => KebabOption;
+
+export type KebabOptionsCreator = (...args: Parameters<KebabAction>) => KebabOption[];
```

Comment:
As suggested in my other comment, I'd prefer to put this type next to React component that actually uses it, i.e. in `public/components/utils/headings.tsx`.

Reply:
ok, moved to `headings.tsx`